jmclient/jmclient/wallet_service.py

```diff
@@ -729,6 +729,15 @@ def get_transaction(self, txid):
     def get_block_height(self, blockhash):
         return self.bci.get_block_height(blockhash)
 
+    def rescanblockchain(self, start_height, end_height=None):
+        return self.bci.rescanblockchain(start_height, end_height)
```

Comment:
`Thread.start()` does not return anything or am I wrong?
```suggestion
        self.bci.rescanblockchain(start_height, end_height)
```

Reply:
Technical answer to your question is to avoid it, we are not *returning* the result of `Thread.start()` anyway :)  Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1461/commits/7f4eaa9846f2f489480de1f2f8bfa9a1e2e96686